addon/serializers/linked-node.js

```diff
@@ -0,0 +1,14 @@
+import OsfSerializer from './osf-serializer';
+
+export default OsfSerializer.extend({
+    serializeIntoHash(hash, typeClass, snapshot, options) {
+        if (options.forRelationship) {
+            hash.data = [{
```

Comment:
I would think the data would need to be overridden - I mimicked Sam's changes in `addon/serializers/institution.js`.  The request for adding an institution to a node, or adding a linked-node to a collection look the same.

URL:          `/v2/collections/{collection_id}/relationships/linked_nodes/`
``` 
  {
                 "data": [{
                   "type": "linked_nodes",   # required
                   "id": <node_id>   # required
                 }]
               }
```